crypto_unit_test.go

```diff
@@ -35,6 +36,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const _Ed25519PubKeyPrefix = "302a300506032b6570032100"
+const _ECDSAPubKeyPrefix = "3036301006072a8648ce3d020106052b8104000a0322000"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏The issue states that the prefix for ECDSA Public key should be `302d300706052b8104000a032200`, so is the issue outdated or is this value not expected?

Reply:
Yes, the ticket is wrong. What we currently have in this PR is aligned with openSSL, which we are currently thinking are following the proper standard for DER.